kong/dao/cassandra/base_dao.lua

```diff
@@ -553,9 +553,12 @@ end
 -- @return `res`
 -- @return `err`
 -- @return `filtering`   A boolean indicating if ALLOW FILTERING was needed by the query
-function BaseDao:count_by_keys(where_t)
+function BaseDao:count_by_keys(where_t, page_size, paging_state)
   local select_q, where_columns, filtering = query_builder.count(self._table, where_t, self._column_family_details)
-  local res, err = self:execute(select_q, where_columns, where_t, {})
+  local res, err = self:execute(select_q, where_columns, where_t, {
+    page_size = page_size,
+    paging_state = paging_state
```

Comment:
Check where it's being used. This avoids showing a `next` button when it's not required, and since the offset is an UUID and not a number, it's not possible to implement this by simply calculating `if (size * offset) > total_count then do_not_show_next() end`.